StudCity.Frontend/src/pages/RegistrationComplete/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from "react";
+import Stepper from "../../components/Stepper/Stepper";
+import UserInformation from "./steps/UserInformation";
+import { useSelector, useDispatch } from "react-redux";
+import {
+	selectActiveStep,
+	changeActiveState,
+	selectRegisterCompleteDefaultInfo,
+	changeRegisterCompleteState
+} from "../../app/features/register-complete/registerCompleteSlice";
+import StepperControll from "../../components/Stepper/StepperControll";
+import Container from "../../components/Container";
+import { getRegisterCompleteDefautlData, registerCompleteSteps } from "../../utils/constants";
+import { registerCompleteUserInformationValidator } from "../../utils/validators/registerCompleteValidators";
+
+
+const RegistrationComplete = () => {
+	const dispatch = useDispatch();
+
+	const activeStep = useSelector(selectActiveStep);
+	const defaultData = useSelector(selectRegisterCompleteDefaultInfo);
+
+	const [valid, setValid] = useState(false);
+	const [state, setState] = useState(getRegisterCompleteDefautlData(defaultData));
+
+	useEffect(() => {
+		setValid(stepValidator());
+	}, [state])
+
+	const handleNext = () => {
+		dispatch(changeRegisterCompleteState(state));
+		if (activeStep === 5) {
```

Comment:
maybe it's a good idea to put 5 into constants folder as I'm seeing a lot of usage of it, consider replacing 5 with a constant in all places where it's used